src/index.json

```diff
@@ -975,6 +975,52 @@
                     "version": "0.1.0"
                 }
             }
+        ],
+        "saphana": [
```

Comment:
`sap-hana` for readability. It is also more consistent with the majority of our other extension names.

Reply:
when I used "sap-hana" as the name in our setup.py, it generated sap_hana which seemed to be rejected by the CI

Reply:
done